modules/ossm-rn-known-issues.adoc

```diff
@@ -34,9 +34,7 @@ These are the known issues in {ProductName}:
 
 * link:https://issues.redhat.com/browse/OSSM-285[OSSM-285] When trying to access the Kiali console, receive the following error message "Error trying to get OAuth Metadata". The workaround is to restart the Kiali pod.
 
-* link:https://issues.jboss.org/browse/MAISTRA-1947[MAISTRA-1947] _Technology Preview_  Updates to ServiceMeshExtensions are not applied.  The workaround is to remove and recreate the ServiceMeshExtensions.
-
-* link:https://issues.jboss.org/browse/MAISTRA-1621[MAISTRA-1621]  _Migration to 2.0_  Prometheus scraping (`spec.addons.prometheus.scrape` set to `true`) does not work when mTLS is enabled.  Additionally, Kiali displays extraneous graph data when mTLS is disabled.
+* link:https://issues.jboss.org/browse/MAISTRA-1959[MAISTRA-1959]  _Migration to 2.0_  Prometheus scraping (`spec.addons.prometheus.scrape` set to `true`) does not work when mTLS is enabled.  Additionally, Kiali displays extraneous graph data when mTLS is disabled.
 +
 This problem can be addressed by excluding port 15020 from proxy configuration, for example,
```

Comment:
```suggestion
Workaround: You can exclude port 15020 from proxy configuration. For example:
```

Reply:
This suggestion is based on the recently-added "Release Notes" section in the Supplemental Style Guide: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes